src/honey_badger/batch.rs

```diff
@@ -9,6 +9,15 @@ pub struct Batch<C, N> {
     pub contributions: BTreeMap<N, C>,
 }
 
+impl<C, N: NodeIdT> Epoched for Batch<C, N> {
+    type Epoch = u64;
+
+    /// Returns the **next** `HoneyBadger` epoch after the sequential epoch of the batch.
+    fn epoch(&self) -> u64 {
+        self.epoch + 1
```

Comment:
I find that very counterintuitive. Should there maybe be a `next_epoch` method in `SenderQueueableOutput` instead?